fastlane/Fastfile

```diff
@@ -212,21 +222,56 @@ end
 after_all do |lane|
   next unless is_ci?
 
+if lane == :beta then
```

Comment:
Wat betreft beta weet ik het niet goed want enkel bij beta heb je een hockey link. In Brutex hebben we dit voor alles en dan staat er gewoon een leeg veld. Vond dat niet zo proper en heb het daarom uit de template gehaald.